http-gateway/service/httpApi.go

```diff
@@ -112,7 +112,7 @@ func resourceEventsMatcher(r *http.Request, rm *router.RouteMatch) bool {
 }
 
 // NewHTTP returns HTTP server
```

Comment:
It doesn't seem like HTTP **server** anymore.

Reply:
fixed